mdata/cache/accnt/accnt_test.go

```diff
@@ -0,0 +1,128 @@
+package accnt
+
+import (
+	"testing"
+	"time"
+)
+
+func getNew(maxSize int) *FlatAccnt {
+	a := &FlatAccnt{
+		total:       0,
+		metrics:     make(map[string]*FlatAccntMet),
+		maxSize:     uint64(maxSize),
+		lru:         NewLRU(),
+		evictQ:      make(chan *EvictTarget, 10),
```

Comment:
any particular reason the unit tests need a custom evictQSize and eventQSize? this would be nicer if we could just call `NewFlatAccount()` (potentially with a parameter to control the stats ticker)